orchestration/db/api.py

```diff
@@ -36,6 +37,42 @@ def session_scope():
         session.close()
 
 
+# -------------------------data access for service definition and
+#                           workflow definition associations------------------
```

Comment:
nit: this commenting style looks weird.  Usually there is "#----" in its own line in the beginning and "#----" in its own line at the end, but not mixed like this.  So it is easier to just remove ----

Reply:
Removed the lines.